omnibus/resources/agent/msi/cal/customaction.h

```diff
@@ -3,8 +3,8 @@
 #define MAX_PASS_LEN 18
 // usercreate.cpp
 bool generatePassword(wchar_t* passbuf, int passbuflen);
-int doCreateUser(const std::wstring& name, const wchar_t * domain, std::wstring& comment, const wchar_t* passbuf);
-int doSetUserPassword(const std::wstring& name, const wchar_t * domain, const wchar_t* passbuf);
+int doCreateUser(const std::wstring& name, std::wstring& comment, const wchar_t* passbuf);
```

Comment:
fixed comment.

can't make passbuf a `std::wstring` because it's (intentionally) not a `std::wstring`. In the cases we generate the password, it's an actual character buffer. We could copy it to a `std::wstring`, but that would ruin the safety code at the end where we zero out the buffer to ensure that the password isn't left in memory.